components/Header/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import CONFIG from '@/lib/configs';
+import { Flex } from '@/styles/containers';
+import { useAuth } from '@/utils/AuthProvider';
+import { useProfile } from '@/utils/ProfileProvider';
+import Icon from '../Icon';
+import { Container, HamburgerButton } from './styles';
+
+interface HeaderProps {
+  toggleNavColumn: () => void;
+}
+
+export default function Header({ toggleNavColumn }: HeaderProps) {
+  const currentPath = usePathname();
+  const { profileReady, profileData } = useProfile();
+  const { userId } = useAuth();
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  useEffect(() => {
+    setIsLoggedIn(userId !== null);
+  }, [userId]);
+
+  const onNavColumnClick = () => {
+    toggleNavColumn();
+  };
+
+  if (currentPath === '/onboarding') {
+    return null;
+  }
+
+  return (
+    <Container>
+      <HamburgerButton onClick={onNavColumnClick}>
```

Comment:
we might want to eventually consolidate all button styles and include them in components/Button.tsx. in this case, we could use a button that basically has no background, outline etc.